packages/big-design/src/components/Flex/types.ts

```diff
@@ -25,8 +25,22 @@ type AlignSelf = ResponsiveProp<
   'auto' | 'normal' | 'self-start' | 'self-end' | 'flex-start' | 'flex-end' | 'center' | 'baseline' | 'stretch'
 >;
 
+type FlexBasis = ResponsiveProp<'auto' | 'fill' | 'min-content' | 'max-content' | 'fit-content' | 'content' | string>;
+
+type FlexColumnGap = ResponsiveProp<string>;
+
 type FlexDirection = ResponsiveProp<'row' | 'column' | 'row-reverse' | 'column-reverse'>;
 
+type FlexGap = ResponsiveProp<string>;
```

Comment:
🤔 why aren't we using a `Spacing` prop for this?

Reply:
Good point, I think I added it because I was following the css api, but makes sense that we should follow BD spacing values. I'll create an issue later today about this. Thanks for the feedback!!